ERROR - 2022-08-26 00:15:21 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(62): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 00:15:59 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(62): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 00:18:25 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(60): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 00:19:09 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(60): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 00:20:39 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 00:21:12 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 00:27:53 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(62): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:13:39 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:13:43 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:19:46 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:20:02 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:48:01 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:53:43 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(61): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:55:15 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(63): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-08-26 21:58:11 --> mysqli_sql_exception: Unknown column 'position' in 'field list' in C:\laragon\www\csm\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\laragon\www\csm\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\csm\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\csm\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\csm\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\csm\Modules\Users\Controllers\UsersController.php(63): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\csm\system\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->add_user()
#6 C:\laragon\www\csm\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
#7 C:\laragon\www\csm\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\csm\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-08-26 23:46:31 --> Too few arguments to function Modules\Users\Controllers\UsersController::get_user(), 0 passed in C:\laragon\www\csm\system\CodeIgniter.php on line 898 and exactly 1 expected
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 69.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:46:48 --> Too few arguments to function Modules\Users\Controllers\UsersController::get_user(), 0 passed in C:\laragon\www\csm\system\CodeIgniter.php on line 898 and exactly 1 expected
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 69.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:47:30 --> Undefined variable: id
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 71.
 1 ROOTPATH\Modules\Users\Controllers\UsersController.php(71): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: id', 'ROOTPATH\\Modules\\Users\\Controllers\\UsersController.php', 71, [])
 2 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:48:14 --> Undefined variable: id
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 71.
 1 ROOTPATH\Modules\Users\Controllers\UsersController.php(71): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: id', 'ROOTPATH\\Modules\\Users\\Controllers\\UsersController.php', 71, [])
 2 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:51:22 --> Call to undefined method Modules\Users\Models\UsersModel::get_data
in SYSTEMPATH\Model.php on line 773.
 1 ROOTPATH\Modules\Users\Controllers\UsersController.php(73): CodeIgniter\Model->__call('get_data', [...])
 2 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:51:51 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::getResult()
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 73.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:52:07 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::getResult()
in ROOTPATH\Modules\Users\Models\UsersModel.php on line 47.
 1 ROOTPATH\Modules\Users\Controllers\UsersController.php(73): Modules\Users\Models\UsersModel->get_user_data('1')
 2 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:53:13 --> Call to a member function getResult() on array
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 73.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:57:21 --> Array to string conversion
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 75.
 1 ROOTPATH\Modules\Users\Controllers\UsersController.php(75): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string conversion', 'ROOTPATH\\Modules\\Users\\Controllers\\UsersController.php', 75, [...])
 2 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:57:38 --> Object of class stdClass could not be converted to string
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 75.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:58:01 --> Object of class stdClass could not be converted to string
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 75.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-08-26 23:58:52 --> Object of class stdClass could not be converted to string
in ROOTPATH\Modules\Users\Controllers\UsersController.php on line 75.
 1 SYSTEMPATH\CodeIgniter.php(898): Modules\Users\Controllers\UsersController->get_user()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Users\Controllers\UsersController))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
